manifest/__init__.py

```diff
@@ -1,2 +1,2 @@
 from .models import *
-from .services import get_manifest_service, ManifestServiceException
\ No newline at end of file
+from .services import get_manifest_service, ManifestServiceException, ManifestService
```

Comment:
A look into the other files' comments, it may render this export not really needed